SDCTA/.eslintrc.json

```diff
@@ -0,0 +1,30 @@
+{
+  "env": {
+    "browser": true,
+    "es2021": true
+  },
+  "extends": ["plugin:react/recommended", "standard-with-typescript"],
+  "overrides": [],
+  "parserOptions": {
+    "ecmaVersion": "latest",
+    "sourceType": "module",
+    "project": "./tsconfig.json"
+  },
+  "plugins": ["react"],
```

Comment:
Honestly don't know what the correct rules to ignore are for the frontend, so I'll leave that to someone working on a frontend team.